test/fixtures/output/kafka/link/list-link-with-tasks.golden

```diff
@@ -0,0 +1,15 @@
+      Task Name      |     State      |             Errors              
+---------------------+----------------+---------------------------------
+  AclSync            | IN_ERROR       | Error Code:                     
+                     |                | AUTHENTICATION_ERROR Error      
+                     |                | Message: Auth issue.,Error      
+                     |                | Code: MISCONFIGURATION_ERROR    
+                     |                | Error Message: Wrong config.    
```

Comment:
Labels like "Error Code:" make this output verbose, and we should add a space after the comma. We could rewrite this to look something like:
```
(AUTHENTICATION_ERROR) Auth issue., (MISCONFIGURATION_ERROR) Wrong config.
```
It would also be great to remove periods in the backend, or here in the frontend if that's not possible.

Reply:
I like:
```
(AUTHENTICATION_ERROR) Auth issue., (MISCONFIGURATION_ERROR) Wrong config.
```
I'm not sure about removing periods here - it feels a bit hacky to mutate the backend error message. I could enclose the message in double quotes?

Reply:
Added to the human output. Do you think we should do the same for the serialized output?

Reply:
Nvm I dont think so. Already have quotes around in the serialized output.